diff --git a/dist/app/composables/manifest.js b/dist/app/composables/manifest.js
index eac5da487476f99a09ddd8cbd5dfd2f51e8095ba..6e19d611d3421257251752e4f7acc972fb8092f7 100644
--- a/dist/app/composables/manifest.js
+++ b/dist/app/composables/manifest.js
@@ -10,11 +10,7 @@ function fetchManifest() {
     throw new Error("[nuxt] app manifest should be enabled with `experimental.appManifest`");
   }
   if (import.meta.server) {
-    manifest = import(
-      /* webpackIgnore: true */
-      /* @vite-ignore */
-      "#app-manifest"
-    );
+    manifest = Promise.resolve({});
   } else {
     manifest = $fetch(buildAssetsURL(`builds/meta/${useRuntimeConfig().app.buildId}.json`), {
       responseType: "json"
